### PR TITLE
Controller-Runtime: Automatically apply milestones

### PR DESCRIPTION
**Pull request by alvaroaleman**
/assign @vincepri

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -357,6 +357,10 @@ milestone_applier:
     master: v0.4
     release-0.4: v0.4
     release-0.3: v0.3
+  kubernetes-sigs/controller-runtime:
+    master: v0.9.x
+    release-0.8: v0.8.x
+    release-0.7: v0.7.x
```
Comment (vincepri) [inline config/prow/plugins.yaml:363]: Let's add 0.6 and 0.5? Or nah?

Comment (alvaroaleman) [reply config/prow/plugins.yaml:363]: Do we still consider them supported?

Comment (vincepri) [reply config/prow/plugins.yaml:363]: Probably only for important security fixes

Comment (alvaroaleman) [reply config/prow/plugins.yaml:363]: updated it

**Comment by alvaroaleman** [comment]
/assign @chaodaiG 
Both vince and me are root-level approvers for controller-runtime: https://github.com/kubernetes-sigs/controller-runtime/blob/master/OWNERS_ALIASES

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**